Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build stage
+FROM rust:1.75 as builder
```

Comment:
Goose generated this file, I'm fiddling with it as it appears there are some other deps when attempting to retrieve credentials